packages/create-react-app/createReactApp.js

```diff
@@ -311,23 +311,24 @@ function createApp(
     }
   }
 
-  if (useYarn) {
-    let yarnUsesDefaultRegistry = true;
-    try {
-      yarnUsesDefaultRegistry =
-        execSync('yarnpkg config get registry')
-          .toString()
-          .trim() === 'https://registry.yarnpkg.com';
-    } catch (e) {
-      // ignore
-    }
-    if (yarnUsesDefaultRegistry) {
-      fs.copySync(
-        require.resolve('./yarn.lock.cached'),
-        path.join(root, 'yarn.lock')
-      );
-    }
-  }
+  // Prevent the cached yarn.lock from being copied over as it fails out build
```

Comment:
This no longer required?

Reply:
Well we don't use yarn so this still valid

Reply:
This was related to travis failing

Reply:
Undone this as we won't use travis